farm/evaluation/metrics.py

```diff
@@ -98,6 +111,11 @@ def compute_metrics(metric, preds, labels):
         return text_similarity_metric(preds, labels)
     # elif metric == "masked_accuracy":
     #     return simple_accuracy(preds, labels, ignore=-1)
+    elif isinstance(metric, list):
+        ret = {}
+        for m in metric:
+            ret.update(compute_metrics(m, preds, labels))
```

Comment:
nice one! recursive call covering all cases, even registered metrics or list of lists.